fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreBase.java

```diff
@@ -685,6 +685,15 @@ default CompletableFuture<FDBStoredRecord<M>> loadRecordAsync(@Nonnull final Tup
     @Nonnull
     CompletableFuture<Void> preloadRecordAsync(@Nonnull Tuple primaryKey);
 
+    /**
+     * Load a {@link FDBSyntheticRecord synthetic record} by loading its stored constituent records and synthesizing it from them.
+     * @param primaryKey the primary key of the synthetic record, which includes the primary keys of the constituents
+     * @return a future which completes to the synthesized record
+     */
+    @Nonnull
+    @API(API.Status.EXPERIMENTAL)
+    CompletableFuture<FDBSyntheticRecord> loadSyntheticRecord(@Nonnull Tuple primaryKey);
```

Comment:
Is the `FDBRecordStore` implementation missing from the PR right now? I couldn't find it, but the PR still compiled, which surprised me.

Reply:
it's implemented in `FDBRecordStore`

Reply:
done.